### PR TITLE
update filesystem prerequisistes

### PR DESCRIPTION
**Pull request by dragonchaser**
This PR adds a missing prerequisites for the filesystem. 

@mmattel feel free to rephrase

### CODE REVIEW

File: modules/ROOT/pages/prerequisites/prerequisites.adoc [hunk 1]
```diff
@@ -273,6 +273,7 @@ ____
 == Filesystems and Shared Storage
 
 Infinite Scale defines drivers for filesystems to store blobs and metadata. The drivers can be configured via the xref:{s-path}/storage-users.adoc[Storage-Users Service Configuration].
+The used filesystem must fully support utf-8 filenames and must be POSIX-compliant.
```
Comment (wkloucek) [inline modules/ROOT/pages/prerequisites/prerequisites.adoc:276]: what happens when one decides to use unicode file names? :thinking:

**Comment by dj4oC** [comment]
utf-8 is enabled on that storage

File: modules/ROOT/pages/prerequisites/prerequisites.adoc (hunk 1 above)
Comment (dragonchaser) [reply modules/ROOT/pages/prerequisites/prerequisites.adoc:276]: Files get lost if the storage decides to `normalize` the filenames to iso-xxxx-xx ....

Comment (dragonchaser) [reply modules/ROOT/pages/prerequisites/prerequisites.adoc:276]: Yes

**Review by butonic — APPROVED** [review]

**Comment by micbar** [comment]
already done in #701

**Pull request closed without merging**